torch/csrc/jit/codegen/cuda/shape_inference.cpp

```diff
@@ -157,6 +157,20 @@ class NaiveTypePropagator {
         node->output()->setType(out_type);
         break;
       }
+      case aten::native_layer_norm: {
+        auto out_type = node->input(0)->type()->cast<TensorType>();
+        node->output(0)->setType(out_type);
+        node->output(1)->setType(out_type);
```

Comment:
This is wrong. stats output from `native_layer_norm` doesn't have the same shape as with data output.

Reply:
Then how do I specify the shape for the stats.